mmv1/products/vmwareengine/NetworkPeering.yaml

```diff
@@ -0,0 +1,177 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'NetworkPeering'
+base_url: 'projects/{{project}}/locations/global/networkPeering'
+self_link: 'projects/{{project}}/locations/global/networkPeerings/{{name}}'
+create_url: 'projects/{{project}}/locations/global/networkPeerings?networkPeeringId={{name}}'
+update_verb: :PATCH
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/compute/docs/reference/rest/v1/networks/addPeering'
+description: |
+  Represents a network peering resource. Network peerings are global resources.
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+
+import_format: ["projects/{{project}}/locations/global/networkPeerings/{{name}}"]
+autogen_async: true
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vmware_engine_network_peering_ven"
+    primary_resource_id: "vmw-engine-network-peering"
+    vars:
+      name: "sample-network-peering"
+    test_env_vars:
+      region: :REGION
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vmware_engine_network_peering_standard"
+    primary_resource_id: "vmw-engine-network-peering"
+    vars:
+      name: "sample-network-peering"
+    test_env_vars:
+      region: :REGION
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The ID of the Network Peering.
+
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Creation time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and
+      up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Last updated time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+
+  - !ruby/object:Api::Type::String
+    name: 'peerNetwork'
+    required: true
+    description: |
+      The relative resource name of the network to peer with a standard VMware Engine network.
+      The provided network can be a consumer VPC network or another standard VMware Engine network.
+
+  - !ruby/object:Api::Type::Boolean
+    name: exportCustomRoutes
+    default_value: true
+    send_empty_value: true
```

Comment:
Any reason for send empty value here? Shouldn't be needed since there is a default value. Presumably there will never be an empty value. 

Same with `importCustomRoutes`, `exportCustomRoutesWithPublicIp`.

Reply:
For these bool fields, the default value should be `true` if not set by the user. However, the user also needs to be able to set it as `false` (which was not being sent to the api without setting send_empty_value as true).